gulpfile.js

```diff
@@ -141,8 +130,14 @@ var miniBuild = function() {
 };
 
 var style = function() {
-  return src('src/style/main.css')
-    .pipe(gHash())
+  var chain = src('src/style/*.css')
+    .pipe(concat('main.css'));
+
+  if (process.env.NODE_ENV === 'production') {
```

Comment:
ah yeah I remember we set this in the miniBuild function. its a bit less direct than the config param earlier but this works

Reply:
`buildIndex` is called in `fastBuild`(`ifyBuild` don't set `process.env.NODE_ENV`) and `build`(`miniBuild` set `process.env.NODE_ENV` -> `'production'`)